File: Bussiness Logic (Tier II)/CompareIT_API/Model/Wish.cs
namespace CompareIT_API.Model
{
    public class Wish
    {
        public string URL { get; set; }

        public Wish(string URL)
        {
            this.URL = URL;
        }

        public string ToString()
        {
            return $"URL: {URL}";
        }
    }
}

File: Database_Java_V7/src/domain/Wish.java
package domain;

public class Wish {
    private String URL;

    public Wish(String URL) {
        this.URL = URL;
    }

    public String getUrl() {
        return URL;
    }

    public void setUrl(String url) {
        this.URL = url;
    }

    @Override
    public String toString() {
        return "Wish{" +
                "url='" + URL + '\'' +
                '}';
    }
}
